client/src/pages/Dashboard/components/ModalReact.js

```diff
@@ -16,15 +18,18 @@ function ModalReact({ setOpenModal, intentName, intentFrequency, intentAssociate
 
       <Modal.Footer>
         <Button variant="secondary" onClick={() => {
-              setOpenModal(false);
-            }}>Close</Button>
+          setOpenModal(false);
+        }}>Close</Button>
         <Button variant="primary" onClick={() => {
-              // DIRECT US TO VOICEFLOW HERE
-              // use {intentName}
-            }}>Add Intent to Voiceflow</Button>
+          // DIRECT US TO VOICEFLOW HERE
+          // use {intentName}
+          console.log('click')
+          addBlock('63728e4cdbfce712134dded9', 'kharadhairya2@gmail.com', '1J7w@r@A*k86', intentName)
```

Comment:
Please do not commit with the password

Reply:
I have changed the password for security purposes.

Reply:
OH SHOOT! Thanks for pointing that out Dhairya! Made the changes 👍